experimental/delegate/mlp.mlir

```diff
@@ -0,0 +1,60 @@
+// RUN: iree-compile --iree-preprocessing-transform-spec-filename=%p/mlp_spec.mlir  %s | \
+// RUN: iree-run-module --device=local-sync \
+// RUN:     --executable_plugin=$IREE_BINARY_DIR/runtime/plugins/AMD-AIE-experimental/delegate/mlp_bf16_aie_delegate$IREE_DYLIB_EXT \
+// RUN:     --module=- \
+// RUN:     --function=mlp_invocation \
+// RUN:     --input="256x256xf32=2" \
+// RUN:     --input="256x256xf32=3"
+
```

Comment:
I think for there to be checks in the test you need to pipe the output to FileCHeck.

Reply:
I will remove the test lines, as it's clear they are causing confusion